.github/workflows/ci.yml

```diff
@@ -8,6 +8,42 @@ on:
   pull_request:
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          toolchain: stable
+          profile: minimal
+          components: clippy, rustfmt
+          override: true
+
+      - run: make init-submodules
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: lint-${{ runner.os }}-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('Cargo.lock') }}
```

Comment:
don't you want to glob for *all* cargo.lock files?  Or you're confident you'll only ever have one?

Reply:
This is the only relevant one we have, thanks to the workspace which ensures there's only one. The only other one is in `relay-cabi`, and that'll be removed soon.

Reply:
Would you prefer if I use '**/Cargo.lock'?

Reply:
it feels a little safer and future proof.  there's nothing type-checking this after all ;)

Reply:
Alright, sounds good. It is a bit incorrect since we now prune caches if the C-ABI changes, but that is a fair temporary tradeoff.